### PR TITLE
Update docs landing

### PR DESCRIPTION
**Pull request by PiotrWodecki**
## Description

This adds a complete refactor of docs together with an addition of the new docs landing page

Closes FCE-636

### CODE REVIEW

File: docs/react-native/quick-setup.md [hunk 1]
```diff
@@ -40,25 +38,24 @@ Use your room manager URL to fetch peer token to get a new room:
...
+If you want to use the camera, you must first request permission.
+Check [this `guide`](/react-native/installation#step-2-configure-app-permissions) for more information.
 
 :::
 
 :::warning
 
-Keep in mind that this won't work on iOS Simulator, as Simulator can't access the camera.
-
+Keep in mind that this won't work on iOS Simulator.
```
Comment (mironiasty) [inline docs/react-native/quick-setup.md:58]: Why this change?

File: docs/react-native/quick-setup.md [hunk 2]
```diff
@@ -91,13 +88,11 @@ function StartStreamingButton({
 ### 3. Show other peers
 
 :::note
-
-In order to get peers, you must first join a room. See the steps above.
-
+In order to get peers, you must first join a room. See steps above.
```
Comment (mironiasty) [inline docs/react-native/quick-setup.md:91]: Can you restore these empty lines?

https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

File: docs/react-native/quick-setup.md [hunk 3]
```diff
@@ -75,7 +72,7 @@ function StartStreamingButton({
 
   const startStreaming = useCallback(async () => {
     const response = await fetch(
-      `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?roomName=*roomName*&peerName=*username*`,
+      `https://cloud.fishjam.work/api/v1/connect/*YOUR_ID*/room-manager/${roomName}/users/${userName}`,
```
Comment (mironiasty) [inline docs/react-native/quick-setup.md:75]: 👀

File: docs/introduction.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 0
+---
+
+import QuickNavigation from "@site/src/components/QuickNavigation";
+import { items } from "@site/src/content/cardItems";
+
+# Getting started
```
Comment (mironiasty) [inline docs/introduction.mdx:8]: WDYT about changing title to `Introduction`. From Getting Started I would expect info how can I start implementing, not info about how it works.

File: docs/react-native/quick-setup.md [hunk 5]
```diff
@@ -242,7 +242,3 @@ const styles = StyleSheet.create({
   },
 });
 ```
-
-### More
-
```
Comment (czerwiukk) [inline docs/react-native/quick-setup.md:247]: **note:** I deleted it because it was linking to the same category menu. Users can also navigate to the next page.

**Review by czerwiukk — APPROVED** [review]

File: docs/introduction.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+2. Once you have an account, Fishjam Cloud will assign you a sandbox instance automatically. Sandbox instances come with [**Room Manager**](/room-manager.md) - a simple backend for creating test rooms. Copy the _Fishjam URL_ and _Management Token_ from the developer panel.
+3. Pick a client SDK that you want to use and follow the Quick Start guide. We provide SDKs for [React Native](/react-native/quick-setup.md) and [React](/react/installation.mdx).
+
+<QuickNavigation
+  items={items.filter((item) => item.title.toLowerCase().includes("setup"))}
+/>
+
+## Do you have any running examples?
+
+We are maintaining a simple videoconferencing app called [Videoroom](https://room.fishjam.io/).
```
Comment (mironiasty) [inline docs/introduction.mdx:32]:
```suggestion
We are maintaining a simple video conferencing app called [Videoroom](https://room.fishjam.io/).
```

**Review by mironiasty — APPROVED** [review]
🥔
